File: namsor/ruby/src/GenderFullExample.rb
require "json"
require "namsor_client"

NamsorClient.configure do |config|
    config.api_key["X-API-KEY"] = "YOUR_API_KEY"
end

begin
    response = NamsorClient::PersonalApi.new.gender_full(
        "Keith Haring", # full_name
    )

    p response
rescue NamsorClient::ApiError => e
    puts "Exception when calling PersonalApi#gender_full: #{e}"
end

File: namsor/python/src/GenderFullExample.py
import json
from datetime import date, datetime
from pprint import pprint

from namsor_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"api_key": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    try:
        response = api.PersonalApi(api_client).gender_full(
            full_name="Keith Haring",
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling PersonalApi#gender_full: %s\n" % e)
